src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/generator_action.py

```diff
@@ -65,18 +65,13 @@ async def build(cls, prefix: str, collection: Collection, name: str) -> ForestSe
             hooks={"load": not schema.static_form, "change": ["changeHook"]},
         )
 
-        # TODO: do we want after story #7 to have all the layout in the forestadminschema.json ?
-        # what about  when there is no layout customization
+        # when there is not layout customizations, don't send the 'layout' property
```

Comment:
strange behaviour, if there is no layout component other than the ‘input’ layout, it is deactivated, but if a single ‘true’ layout component is added, it is possible to have several ‘input’ layouts. The ‘input’ element seems useless 🤔